featurecompletions/featurecompletions.go

```diff
@@ -114,7 +114,7 @@ func (f *FeatureCompletion) ApplyToSchema(schema *serializers.Schema, _ migratio
 		if idempotently {
 			return nil
 		}
-		return fmt.Errorf("Couldn't locate feature_completion of %s in schema", *f.featureGate)
+		return fmt.Errorf("couldn't locate feature_completion of %s in schema", *f.featureGate)
```

Comment:
Why not? Should we be using something other than an inline string if we want to return a capital "C" to the end user?

Reply:
It's fine -- generally happy to stick with community conventions in 99.9% of cases.